### PR TITLE
Create README-de_DE

### PR DESCRIPTION
**Pull request by Segelzwerg**
A translation of the readme into German.

### CODE REVIEW

File: README-de_DE [hunk 1]
```diff
@@ -0,0 +1,125 @@
+# truckersmp-cli
+
+truckersmp-cli ist einfach zu benutzendes Skript zum Herunterladen von TruckersMP und Starten des Spieles.
+
+Es installiert die Windowsversiond von American Truck Simulator (`-a`) oder Euro Truck Simulator 2 (`-e`)
```
Comment (Lucki) [inline README-de_DE]:
```suggestion
Es installiert die Windowsversion von American Truck Simulator (`-a`) oder Euro Truck Simulator 2 (`-e`) und
```

File: README-de_DE [hunk 2]
```diff
@@ -0,0 +1,125 @@
...
+updatet mit steamcmd (`-u`). Der Mod wird mit (`-s`) gestartet entweder mit (`-p`) mit Proton aka. Steam Play
```
Comment (Lucki) [inline README-de_DE]:
```suggestion
updatet mit steamcmd (`-u`). Der Mod wird mit (`-s`) gestartet und entweder mit Proton (`-p`) aka. Steam Play
```

File: README-de_DE [hunk 3]
```diff
@@ -0,0 +1,125 @@
...
+oder Wine (`-w`).
```
Comment (Lucki) [inline README-de_DE]:
```suggestion
oder Wine (`-w`) ausgeführt.
```

File: README-de_DE [hunk 4]
```diff
@@ -0,0 +1,125 @@
...
+
+Es braucht eine funktionierende Steam Installation unter `$XDG_DATA_HOME/Steam`, um es mit Proton zu starten
+die Spieldateien zu aktualisieren. Es wird alle Steamprozesse während der Aktualisierung stoppen,
+um zu verhindern, dass Steam nach einem Passwort oder dem Guard Code beim nächsten Start fragt.
+Falls Standard Wine verwendet wird, sollte er st die Windowsversion von Steam gestartet werden.
```
Comment (Lucki) [inline README-de_DE]:
```suggestion
Falls Wine verwendet wird, sollte zuerst die Windowsversion von Steam gestartet werden.
```

File: README-de_DE [hunk 5]
```diff
@@ -0,0 +1,125 @@
+# truckersmp-cli
+
+truckersmp-cli ist einfach zu benutzendes Skript zum Herunterladen von TruckersMP und Starten des Spieles.
+
+Es installiert die Windowsversiond von American Truck Simulator (`-a`) oder Euro Truck Simulator 2 (`-e`)
+updatet mit steamcmd (`-u`). Der Mod wird mit (`-s`) gestartet entweder mit (`-p`) mit Proton aka. Steam Play
+oder Wine (`-w`).
+
+Es braucht eine funktionierende Steam Installation unter `$XDG_DATA_HOME/Steam`, um es mit Proton zu starten
+die Spieldateien zu aktualisieren. Es wird alle Steamprozesse während der Aktualisierung stoppen,
```
Comment (Lucki) [inline README-de_DE]:
```suggestion
und die Spieldateien zu aktualisieren. Es wird alle Steamprozesse während der Aktualisierung stoppen,
```

**Comment by Lucki** [comment]
We've pushed a major update with a lot of new additions. Please revisit your PR to match with the latest additions.

I'm not familiar with multiple readme languages. Will GitHub show them automatically or do we have to set them somewhere?

**Comment by kakurasan** [comment]
> I'm not familiar with multiple readme languages. Will GitHub show them automatically or do we have to set them somewhere?

AFAIK "automatically" is not possible. GitHub only converts Markdown to HTML.
(See https://stackoverflow.com/questions/47457811/add-multiple-readme-on-github-repo)

I would like clickable "Language list" near the top (or bottom) of each page.

Example: [English](https://github.com/lhark/truckersmp-cli/blob/master/README.md) | [Deutsch](https://github.com/lhark/truckersmp-cli/blob/master/README-de_DE.md)

Every README file needs to have the list (Don't forget to add to English `README.md`). Each language name should be "native" (non-English name).

We can use Markdown links:

```markdown
<!-- Add this to README.md (English) and README-*.md (translated) -->
[English](https://github.com/lhark/truckersmp-cli/blob/master/README.md) | [Deutsch](https://github.com/lhark/truckersmp-cli/blob/master/README-de_DE.md)
```

In addition, @Segelzwerg needs to rename the file to `README-de_DE.md` (currently it is *not Markdown but plain text*, `.md` extension is needed).

**Comment by Lucki** [comment]
Because nothing happened in the meantime and there will be another major readme update coming soon I close this PR for now. You should be able to reopen this PR when you've took the most recent changes into account.

In the meantime I can recommend everyone who needs a translation to use [DeepL](https://www.deepl.com/translator) which generates much better translations than Google. There are [add-ons](https://addons.mozilla.org/en-US/firefox/addon/translate-now/) available which make the translation process really convenient.

**Pull request closed without merging**